python/common/org/python/types/FrozenSet.java

```diff
@@ -46,6 +46,34 @@ public FrozenSet(org.python.Object[] args, java.util.Map<java.lang.String, org.p
     @org.python.Method(
             __doc__ = ""
     )
+    public org.python.Object __invert__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary ~: 'frozenset'");
+    }
+
+    @org.python.Method(
+            __doc__ = ""
+    )
+    public org.python.Object __pos__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary +: 'frozenset'");
+    }
+
+    @org.python.Method(
+            __doc__ = ""
+    )
+    public org.python.Object __neg__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary -: 'frozenset'");
+    }
+
+    @org.python.Method(
+            __doc__ = ""
+    )
+    public org.python.types.Bool __bool__() {
+        return new org.python.types.Bool(this.value.size() > 0);
```

Comment:
Could you check `test_bool.BuiltinBoolFunctionTests` and see why it's not passing for frozenset yet?